tests/end_to_end/test_netcat/data/playbooks/configuration.yaml

```diff
@@ -0,0 +1,61 @@
+- name: Test agent configuration
+  hosts: wazuh-agent
+  tasks:
+
+    - name: Configure agent to get a list of running processes
+      become: true
+      blockinfile:
+        path: /var/ossec/etc/ossec.conf
+        insertbefore: </ossec_config>
+        block: |
+          <localfile>
+          <log_format>full_command</log_format>
+          <alias>process list</alias>
+          <command>ps -e -o pid,uname,command</command>
+          <frequency>30</frequency>
+          </localfile>
+        marker: <!-- {mark} ANSIBLE MANAGED BLOCK -->
+
+    - name: Restart wazuh-agent
+      become: true
+      shell: systemctl restart wazuh-agent
+
+    - name: Install netcat (CentOS)
+      become: true
+      shell: yum install -y nmap-ncat
+      when: ansible_facts['distribution'] == "CentOS"
+
+    - name: Install netcat (Ubuntu)
+      become: true
+      shell: apt install ncat nmap -y
+      when: ansible_facts['distribution'] == "Ubuntu"
```

Comment:
It should be noted in the documentation that this test can only be run on Ubuntu and CentOS.